### ISSUE TITLE
Problems with modules loaded with gulp-load-plugins

### ISSUE THREAD

**Issue by gnicol-bzh** [maxogden/dependency-check#44]
Hi,

It seems that modules loaded with `gulp-load-plugins` are not recognized.

Example:
```javascript
var gulp = require('gulp');
var $ = require('gulp-load-plugins')();

gulp.task('eslint', function() {
    return gulp.src('js/**/*.js')
        .pipe($.eslint())
        .pipe($.eslint.format())
        .pipe($.eslint.failAfterError());
});
```

- If `gulp-eslint` is already in `node_modules` but not listed in `package.json` dependencies, it is reported as `unused` instead of `missing`.
- If `gulp-eslint` is not in `node_modules` but listed in `package.json` dependencies, it is not reported as `missing`.

**Comment by voxpelli** [comment]
As you're not explicitly loading the mentioned gulp modules, then there's no way for this module to know whether they are used or not? So I would say that this seems to work as expected, no?

You can however exclude a specific module from the test with the `--ignore-module`, `-i` option: https://github.com/maxogden/dependency-check#--ignore-module--i

You can also ignore all dev-dependencies with the `--no-dev` option if you don't want any verification of them: https://github.com/maxogden/dependency-check#--no-dev

Closing for now, hoping that those two options above will help you enough, otherwise please do comment with the expected behavior and I'll reopen this issue again.
